contracts/src/v0.8/tests/Counter.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+contract Counter {
+  uint256 public count = 0;
+
+  function increment() public returns (uint256) {
+    count += 1;
+    return count;
+  }
+
+  function reset() public returns (uint256) {
+    count = 0;
+    return count;
```

Comment:
nitpick, but does this need to return anything?

Reply:
nope, just copy pasted this :p